pkg/apis/pipelinesascode/register.go

```diff
@@ -21,4 +21,34 @@ const (
 	GroupName       = "pipelinesascode.tekton.dev"
 	RepositoryKind  = "Repository"
 	V1alpha1Version = "v1alpha1"
+
+	// Labels and Annotation Keys
+
+	TaskKey            = GroupName + "/task"
```

Comment:
I kinda *hate* those Key prefix, don't we have enums in go or some sort of struct for that

Reply:
just move all of the const and setting to pipelinesascode/const/ package and we use const.Task const.Pipeline etc.. wdyt?

or have a struc that has everything in there..

Reply:
`pipelinesascode/const/` yeah this make sense.
not sure how struct can be used here 🤔 \
I was confused by `Key prefix` in you comment .. 😅 I think you meant writing them as `pipelinesascode.key`

Reply:
yeah sounds good, i think that refactoring should be easy with goland, lmk if that works for you and i'll merge it asap so we don't have too many conflicts

Reply:
yeah will update... const is a inbuilt keyword
might have to give something different for the pkg